app/models/ability.rb

```diff
@@ -60,6 +62,7 @@ def initialize(user)
       can [:view_applicants, :edit_applicants, :view_participants, :print_applications, :manage, :view_material, :upload_material, :print_agreement_letters, :download_material, :view_unpublished], Event
       can :send_email, Email
       can :manage, Request
+      cannot :apply, Event
```

Comment:
I'm still not sure if we need this line. It should be already inherited from coach.
The problem is the :manage in line 60/62, since :manage means an organizer can do everything with an event.
If this should be in there, I guess we could skip all the other permissions for event